templates/api-template-go/main.go

```diff
@@ -62,10 +62,12 @@ func Handler(ctx context.Context, event map[string]interface{}) (Response, error
 		var payLoad map[string]interface{}
 		// Your GET method should be handled here
 		if (event["method"].(string) == "GET"){
+      logger.VERBOSE("Sample log inside GET");
```

Comment:
change to info as default log level is info, ie only error, warning and info will be printed. Do in other files too

Reply:
Done